### ISSUE TITLE
Fix urls

### ISSUE THREAD

**Issue by romainfrancois** [tidyverse/dplyr#5484]
From cran: 

```
 Found the following (possibly) invalid URLs:
   URL: http://blog.rstudio.org/2016/03/24/tibble-1-0-0/ (moved to
https://blog.rstudio.com/2016/03/24/tibble-1-0-0/)
     From: NEWS.md
     Status: 200
     Message: OK
   URL: http://r-datatable.com/ (moved to
https://rdatatable.gitlab.io/data.table)
     From: README.md
     Status: 200
     Message: OK
   URL: http://r4ds.had.co.nz/transform.html (moved to
https://r4ds.had.co.nz/transform.html)
     From: README.md
     Status: 200
     Message: OK
   URL: https://groups.google.com/group/manipulatr (moved to
https://groups.google.com/d/forum/manipulatr)
     From: README.md
     Status: 200
     Message: OK
```

**Comment by cathblatter** [comment]
corrected the url's in the README.Rmd, I already closed the PR #5486 (from my mobile device).

Also, I fail to locate the url in the News.md file and don't know why: 

```
Found the following (possibly) invalid URLs:
   URL: http://blog.rstudio.org/2016/03/24/tibble-1-0-0/ (moved to
https://blog.rstudio.com/2016/03/24/tibble-1-0-0/)
     From: NEWS.md
     Status: 200
     Message: OK

```
I looked in the News.md file.

**Comment by romainfrancois** [comment]
Thanks. I've reopen the pull request.
